### ISSUE TITLE
Unable to generate bitstream

### ISSUE THREAD

**Issue by climblinne** [Digilent/Arty-Z7-20-hdmi-out#1]
1. The top module name is not set, after generating the project. May be the difference between project name and block diagram name?
2. After setting the top module name, there is still the following problem:

> ERROR: [DRC 23-20] Rule violation (NSTD-1) Unspecified I/O Standard - 7 out of 145 logical ports use I/O standard (IOSTANDARD) value 'DEFAULT', instead of a user assigned specific value. This may cause I/O contention or incompatibility with the board power or connectivity affecting performance, signal integrity or in extreme cases cause damage to the device or the components to which it is connected. To correct this violation, specify all I/O standards. This design will fail to generate a bitstream unless all logical ports have a user specified I/O standard value defined. To allow bitstream creation with unspecified I/O standard values (not recommended), use this command: set_property SEVERITY {Warning} [get_drc_checks NSTD-1].  NOTE: When using the Vivado Runs infrastructure (e.g. launch_runs Tcl command), add this command to a .tcl file and add that file as a pre-hook for write_bitstream step for the implementation run. Problem ports: HDMI_DDC_scl_i, HDMI_DDC_scl_o, HDMI_DDC_scl_t, HDMI_DDC_sda_i, HDMI_DDC_sda_o, HDMI_DDC_sda_t, HDMI_HPD_tri_i[0].

It would be nice, when the samples would run without any users bugfix.
Development environment is Vivado 2016.4.

**Comment by sbobrowicz** [comment]
I couldn't replicate your problem. Please ensure you are following the wiki instructions:

https://reference.digilentinc.com/learn/programmable-logic/tutorials/arty-z7-hdmi-demo/start

Things to be careful of are:

1) that you have cd'ed into the proj directory before running source create_project.tcl
2) that you downloaded the project by cloning the repo or from the "releases" download. The Github "Download ZIP" button does not work properly.
3) That you installed the digilent board files prior to running create_project.tcl.

If you still have issues, please post on our forum and we will get you up and running:

forum.digilentinc.com

**Comment by sbobrowicz** [comment]
Decided to leave this open so you can comment on what went wrong. This is a new vivado project structure we are using and could use feedback for the documentation if something is unclear.

**Comment by climblinne** [comment]
I have missed the not working git submodule call, when I downloaded the sources with git.

The git submodule reference to the "http" instead of "https" address of the vivado library, so it is not able to download the library. Github is not supporting "http" anymore!

Also I used the Vivado Tcl Shell (commandline) instead of the Vivado Shell inside the Vivado UI, which lead to a wrong project file generation.
